### PR TITLE
Upgrade telepresence to 2.16.0

### PR DESCRIPTION
**Pull request by d6e-automaton**
Upgrade telepresence to 2.16.0

### CODE REVIEW

File: emojivoto-web-app/js/setup_dev_env.sh [hunk 1]
```diff
@@ -199,10 +199,10 @@ connect_local_dev_env_to_remote() {
     telepresence helm upgrade
     telepresence login --apikey=${AMBASSADOR_API_KEY}
     telepresence quit -s
-    telepresence connect --docker
+    telepresence connect --docker --context default --namespace ${EMOJIVOTO_NS}
     
     interceptName=$(kubectl get rs -n emojivoto --selector=app=web-app --no-headers -o custom-columns=":metadata.name")
-    telepresence intercept ${interceptName} --docker --context default -n ${EMOJIVOTO_NS} --service web-app --port 8083:80 --ingress-port 80 --ingress-host ${svcName}.ambassador --ingress-l5 ${svcName}.ambassador --preview-url=true
+    telepresence intercept ${interceptName} --service web-app --port 8083:80 --ingress-port 80 --ingress-host ${svcName}.ambassador --ingress-l5 ${svcName}.ambassador --preview-url=true
```
Comment (njayp) [inline emojivoto-web-app/js/setup_dev_env.sh:205]: I think we still want the docker and context flags here, right?

**Review by njayp — CHANGES_REQUESTED** [review]

File: emojivoto-web-app/js/setup_dev_env.sh (hunk 1 above)
Comment (knlambert) [reply emojivoto-web-app/js/setup_dev_env.sh:205]: I wonder if it is mandatory though since we set it for the connect

Comment (knlambert) [reply emojivoto-web-app/js/setup_dev_env.sh:205]: ok no it's deprecated

Connected to context default, namespace emojivoto (https://demo-prod-2a1ffadb-791c-4393-afe5-279a9191ea6a.cluster.bakerstreet.io)
Flag --docker has been deprecated, use telepresence connect to initiate the connection

**Review by knlambert — APPROVED** [review]
Works as expected after the fix of the container name.

**Review by knlambert — APPROVED** [review]

**Pull request merged**